package.json

```diff
@@ -877,7 +877,8 @@
           "default": [
             "--ignore-missing-imports",
             "--follow-imports=silent",
-            "--show-column-numbers"
+            "--show-column-numbers",
+            "--no-pretty"
```

Comment:
I'm not sure how this actually works. But this was the only reference to these flags that I could find...?